ecosystem/resources/members/cala_quantum_library.toml

```
@@ -0,0 +1,12 @@
+name = "CALA-n Quantum Library"
+url = "https://github.com/albayaty/cala_quantum_library"
+description = "A set of cost-effective n-bit gates based on Clifford+T gates, quantum layouts, and n neighboring physical qubits (2 \\<= n \\<= 5)."
```

```suggestion
description = "A set of cost-effective n-bit gates based on Clifford+T gates, quantum layouts, and n neighboring physical qubits (2 <= n <= 5)."
```
Guess this is a bug in our script somewhere